test/arsi_test.rb

```diff
@@ -1,5 +1,5 @@
 require_relative "helper"
-SingleCov.covered!
+SingleCov.covered! uncovered: 2
```

Comment:
@grosser I don’t know why SingleCov complains about two untested lines. If I remove or change either of the two lines it says are untested, the tests fail.

Reply:
usually that's the else case not being covered

basically need 2 tests for this:
```
raise if true
```

```
if true
  puts 1
  raise
else
  puts 2
end
```